### PR TITLE
couple refactoring of swagger-page titles

### PR DESCRIPTION
**Pull request by evgeniimv**

### CODE REVIEW

File: tes-adapter/src/main/java/com/epam/pipeline/tesadapter/configuration/TesSwaggerConfig.java [hunk 1]
```diff
@@ -26,10 +26,8 @@ public Docket api() {
 
     private ApiInfo getApiInfo() {
         return new ApiInfoBuilder()
-                .title("TES adapter API Documentation")
-                .description("Simple TES adapter application. " +
-                        "Without Security Configuration and Context")
-                .version("1.0.0")
+                .title("Task Execution Service (TES) API")
+                .description("Describes the REST API provided by a Cloud-Pipeline Task Execution Service")
```
Comment (SilinPavel) [inline tes-adapter/src/main/java/com/epam/pipeline/tesadapter/configuration/TesSwaggerConfig.java]: let's provide more specific information like:
`Represents implementation of GA4GH TES (https://github.com/ga4gh/task-execution-schemas) API for Cloud Pipeline`

File: tes-adapter/src/main/java/com/epam/pipeline/tesadapter/configuration/TesSwaggerConfig.java [hunk 2]
```diff
@@ -26,10 +26,8 @@ public Docket api() {
 
     private ApiInfo getApiInfo() {
         return new ApiInfoBuilder()
-                .title("TES adapter API Documentation")
-                .description("Simple TES adapter application. " +
-                        "Without Security Configuration and Context")
-                .version("1.0.0")
+                .title("Task Execution Service (TES) API")
```
Comment (SilinPavel) [inline tes-adapter/src/main/java/com/epam/pipeline/tesadapter/configuration/TesSwaggerConfig.java]: Task Execution Service (GA4GH TES) API

File: tes-adapter/src/main/java/com/epam/pipeline/tesadapter/configuration/TesSwaggerConfig.java (hunk 1 above)
Comment (evgeniimv) [reply tes-adapter/src/main/java/com/epam/pipeline/tesadapter/configuration/TesSwaggerConfig.java]: fixed

File: tes-adapter/src/main/java/com/epam/pipeline/tesadapter/configuration/TesSwaggerConfig.java (hunk 2 above)
Comment (evgeniimv) [reply tes-adapter/src/main/java/com/epam/pipeline/tesadapter/configuration/TesSwaggerConfig.java]: fixed

**Pull request merged**